src/com/obelikova/console/calculator/ConsoleCalculator.java

```diff
@@ -0,0 +1,68 @@
+package com.obelikova.console.calculator;
+
+/**
+ * a. Supported operations: +, -, /, *;
+ * b. Number of operands: 2;
+ * c. Input method: command line arguments (operation, operand1, operand2). Note: operands shall be real;
+ * d. Output method: console (operand1 operation operand2 = result, i.e. "2 + 3 = 7" );
+ * e. Unit tests shall cover all operations.
+ */
+
+public class ConsoleCalculator {
+
+  public ConsoleCalculator() {}
+
+  public static void main(String[] args) {
+    double firstArg;
+    double secondArg;
+    double result;
+    String operation = args[1];// What initial value should be?
```

Comment:
How about adding control statement to check that you have been passed the required amount of input arguments and they are of appropriate type? Otherwise, I suggest to inform the user "How to use you application"